### ISSUE TITLE
How to use add_dataframe with simmer.opt

### ISSUE THREAD

**Issue by nguyenbm164** [r-simmer/simmer.optim#16]
Hi, I am BI Engineer in Ecommerce field. 

After researching a lot of packages, I found simmer, specifically it is simmer.optim can meet my expectation, but there is an issue that I don't know how to set up a data frame with predefined arrivals at a specific time, then optimize to find the number of headcounts in each time based on a schedule function and get the capacity. Please help, thank you a lot

**Comment by Enchufa2** [comment]
What is specifically your question/issue here?

- If you are looking for how `add_dataframe` works, the [reference page](https://r-simmer.org/reference/add_dataframe.html) is the proper place to start. Basically, it works in the same way as `add_generator` by default, but you specify a vector of interarrival times in the `time` column of a data frame. But it's also possible to input absolute times by specifying `time="absolute"`.
- If you are looking for usage examples, there's one in the vignettes [here](https://r-simmer.org/articles/simmer-06-queueing.html?q=add_dataframe#state-dependent-service-rates).

So the first step would be to have a working simulation for a given set of parameters. Then the next step is to take a look at `simmer.optim`'s [README](https://github.com/r-simmer/simmer.optim#readme) for examples on how to optimize those parameters. Finally, if you have specific issues in your simulation/optimization, the [mailing list](https://r-simmer.org/#mailing-list) would be the proper place to share your code so far and ask for help.

In other words, let's keep this issue tracker for _issues_ in the package, and not usage questions, please.
